### ISSUE TITLE
possibility to exclude parts of code from reduction

### ISSUE THREAD

**Issue by Trass3r** [csmith-project/creduce#156]
I couldn't find a way to tell creduce to not touch certain parts of the code.
Similar to DustMite's --no-remove "regex" option resp. in-code with a // DustMiteNoRemoveStart comment.

**Comment by eeide** [comment]
You are right: There is no command-line option or source comment that means "don't touch this" to C-Reduce. Thank you for the (implicit) suggestion!

Maybe you can achieve the effect you want by coding it into the script that you give to C-Reduce?

For example, to enforce a rule that there is at least one call to "foo" in the program, grep for "foo()".

**Comment by Trass3r** [comment]
The script trick works for simple cases but does not scale.
